src/components/beetframe.vue

```diff
@@ -13,8 +13,6 @@
 </template>
 <script>
 
-import { EventBus } from '../lib/event-bus.js';
-
```

Comment:
There is a reason this was there ( I've just haven't finished it yet).  Using it to communicate across components and in this case it's going to be used to show a notification "Beet is minimised to the taskbar and will run in the background until you quit" with a "Don't show this again checkbox"

Reply:
reverted.